### PR TITLE
feat(sink): qr code generator

### LINKED ISSUE #627: Kitchen Sink: Build A QR Code Generator:
Build a QR code generator that converts text or URLs into QR codes that you'll add to the kitchen sink. Feel free to use any existing projects of yours or start one from scratch.

For more information about the Kitchen sink; check out:

- https://sink.million.dev/
- https://github.com/aidenybai/million/blob/main/packages/kitchen-sink/README.md

N.B: **Upon PR review, your usage of Million.js is a plus**

### PR DESCRIPTION
**Pull request by aelew**
**Please describe the changes this PR makes and why it should be merged:**
This PR adds a QR code generator to the Kitchen Sink.
Closes #627.

**Status**

- [x] Code changes have been tested against prettier, or there are no code changes
- [x] I know how to update typings and have done so, or typings don't need updating

**Semantic versioning classification:**

- [ ] This PR changes the codebase
  - [ ] This PR includes breaking changes (methods removed or renamed, parameters moved or removed)
  - [ ] This PR changes the internal workings with no modifications to the external API (bug fixes, performance improvements)
- [x] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: packages/kitchen-sink/src/examples/qr-code-generator.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          style={{
+            display: 'inline-flex',
+            background: 'white',
+            marginTop: 32,
+            padding: 16,
+          }}
+        >
+          <QRCode value={data} />
+        </div>
+      ) : null}
```
Comment (deeola) [inline packages/kitchen-sink/src/examples/qr-code-generator.tsx:40]:
```suggestion
     {data && (
        <div
          style={{
            display: 'inline-flex',
            background: 'white',
            marginTop: 32,
            padding: 16,
          }}
        >
          <QRCode value={data} />
        </div>
      )}
```

File: packages/kitchen-sink/src/examples/qr-code-generator.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        </button>
+      </div>
+      {data ? (
+        <div
+          style={{
+            display: 'inline-flex',
+            background: 'white',
+            marginTop: 32,
+            padding: 16,
+          }}
```
Comment (deeola) [inline packages/kitchen-sink/src/examples/qr-code-generator.tsx:36]: Hi, is it possible to have the styles somewhere else?

Comment (aelew) [reply packages/kitchen-sink/src/examples/qr-code-generator.tsx]: 👍

**Review by deeola — APPROVED** [review]

**Comment by aidenybai** [comment]
Thank you for the contribution @aelew!

**Pull request merged**